nextflow.config

```diff
@@ -282,22 +310,19 @@ validation {
 \033[0;34m  |\\ | |__  __ /  ` /  \\ |__) |__         \033[0;33m}  {\033[0m
 \033[0;34m  | \\| |       \\__, \\__/ |  \\ |___     \033[0;32m\\`-._,-`-,\033[0m
                                         \033[0;32m`._,._,\'\033[0m
-\033[0;35m  ${manifest.name} ${manifest.version}\033[0m
+\033[0;35m  nf-core/multiplesequencealign ${manifest.version}\033[0m
 -\033[2m----------------------------------------------------\033[0m-
 """
-        afterText = """${manifest.doi ? "* The pipeline\n" : ""}${manifest.doi.tokenize(",").collect { "  https://doi.org/${it.trim().replace('https://doi.org/','')}"}.join("\n")}${manifest.doi ? "\n" : ""}
+        afterText = """${manifest.doi ? "\n* The pipeline\n" : ""}${manifest.doi.tokenize(",").collect { "    https://doi.org/${it.trim().replace('https://doi.org/','')}"}.join("\n")}${manifest.doi ? "\n" : ""}
 * The nf-core framework
     https://doi.org/10.1038/s41587-020-0439-x
 
 * Software dependencies
-    https://github.com/${manifest.name}/blob/master/CITATIONS.md
+    https://github.com/nf-core/multiplesequencealign/blob/master/CITATIONS.md
 """
     }
     summary {
         beforeText = validation.help.beforeText
         afterText = validation.help.afterText
     }
 }
-
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
```

Comment:
there's an error with this change, don't remove it! We are going to have a patch release soon to fix it

Reply:
Added!